packages/cdc-react/src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { cleanup, render } from "@testing-library/react";
+import { afterEach } from "vitest";
+import { Card } from "./Card";
+
+afterEach(() => {
+  cleanup();
+});
+
+describe("Card component", () => {
+  it("should render the card component", () => {
+    render(<Card cardLayout="horizontal" cardAction="test" />);
```

Comment:
Need to actually assert against some expected value.  For this test, you can just assert that an element with text "test" is in the document.